webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -56,6 +70,24 @@ class SubscriptionsPage extends Component {
       if (!prevProps.organization || prevProps.organization.id !== organization.id) {
         this.loadData();
       }
+
+      if (disconnected === false && disconnected !== prevProps.settings.disconnected) {
```

Comment:
any reason to not use `!disconnected` here? `if (!disconnected && disconnected !== prevProps.settings.disconnected)`

Reply:
When I wrote it that way I saw that disconnected was coming through as undefined (i didn't want the conditional to evaluate true when disconnected changed from undefined to false). I also had to rebase this PR on some recent changes you made which changed how the disconnected setting got loaded and where in the props it came from. I also have to cherry-pick this back into 3.16 which has the 'old' disconnected prop. For those reasons I chose to be as explicit as possible.

In order for disconnected to not start as `undefined` I think it needs a default state set since it's coming from redux and the default prop is basically ignored, is that right? If so, I'm hesitant to make that change here since I want to avoid more merge conflicts when I pick it to 3.16

Bet you weren't expecting such a long answer to this question! :smile:

Reply:
> In order for disconnected to not start as `undefined` I think it needs a default state set since it's coming from redux and the default prop is basically ignored, is that right?

Ya that sounds likely.

Ok, that is fine, I don't see an issue checking it explicitly, we care about a boolean false value rather than a falsey one. Thanks for the explanation! :+1:

Reply:
@jturel Default `prop-types` will get set on the component regardless of Redux, so they wouldn't be ignored.

1. When making the connected component, Redux runs `mapStateToProps` and assigns props from the Redux store accordingly.
2.  If one of those values from the Redux store isn't defined, the component won't receive that prop, and so will look to `prop-types`.
3.  If `prop-types` has a default PropType value for that prop, it will get assigned to the component. 

Even so I don't see any harm in being explicit with `disconnected === false`.

Reply:
that makes sense. weird though - i definitely saw that `disconnected` was undefined. at least it works fine regardless :)